build.sbt

```diff
@@ -102,6 +104,13 @@ lazy val validation = (project in file("validation"))
 lazy val panel = (project in file("panel"))
   .settings(hmdaBuildSettings: _*)
   .settings(
+    assemblyMergeStrategy in assembly := {
+      case "application.conf" => MergeStrategy.concat
+      case "META-INF/io.netty.versions.properties" => MergeStrategy.concat
+      case x =>
+        val oldStrategy = (assemblyMergeStrategy in assembly).value
```

Comment:
I think I have a general idea of what this block does (and the others that are like it in this file).  Here's my understanding, please correct what I've gotten wrong 😁 : 

I think, from reading the [sbt-assembly](https://github.com/sbt/sbt-assembly) documentation, that it is determining what to do with filepaths that are duplicated in multiple projects while assembling the `.jar`.  (e.g. `panel/src/main/resources/application.conf` has the same relative filepath as `persistence-model/src/main/resources/application.conf`, so if the `MergeStrategy` is `concat`, it will append one onto the other and use the resulting file.)

But I'm confused by this line. Is it simply telling this project (`panel`) to use the same `assemblyMergeStrategy` as the main `hmda` project for any filepath that doesn't match the cases on lines 108 or 109?